.github/workflows/dockerhub.yml

```diff
@@ -13,23 +13,19 @@ jobs:
       - name: Check out the repo
         uses: actions/checkout@v3
       - name: Log in to Docker Hub
-        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
       - name: Extract metadata (tags, labels) for Docker
         id: meta
-        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        uses: docker/metadata-action@v4
         with:
-          images: my-docker-hub-namespace/my-docker-hub-repository
+          images: permanentlegacy/my-docker-hub-repository
```

Comment:
I am waiting for @cecilia-donnelly to give me access so I can get the right tag for this line

Reply:
Based on our slack conversation and the back-end pr I thought we weren't pushing this image to dockerhub.

Reply:
@cecilia-donnelly That's was one of the options I listed. The other one was to publish publicly without the secrets.

I don't mind removing the publish action! So, I would just do that.

Reply:
Thank you!